### DISCUSSION TITLE
New Contributor

### DISCUSSION THREAD

**Discussion by ssoufi-code** [hyprwm/Hyprland#4642] [General]
Hi, I am a  senior college student who has a background with coding. I have taken a couple of intro classes in C/C++ a few years  ago, but most of my experience is with Python. I have been using Linux as my daily driver for over 2 years now, though mainly on Gnome.  As a result, I am completely new to the world of tiling window managers. If I am going to contribute, I will need a few project ideas, or easy problems to learn more about developing for Hyprland and C++. Do you have any of them? A second but related question, is should I be contributing in the first place, because I don't have the requisite background

**Comment by vaxerski** [comment]
this is foss, everyone can make a mr if they feel like doing so. I am not a tutor, though.
